client-app/shared/account/components/sign-in-form.vue

```diff
@@ -78,6 +82,8 @@ import { useCart } from "@/shared/cart";
 import { mergeCart } from "@/xapi/graphql/cart";
 import { eagerComputed } from "@vueuse/core";
 
+const USER_IS_LOCKED_OUT_ERROR_CODE = "user_is_locked_out";
```

Comment:
It's better to use `enum` here to avoid too much constants if other error handlers appear.